media-gfx/yafaray/yafaray-3.3.0.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6} )
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="A free open-source montecarlo raytracing engine"
+HOMEPAGE="http://www.yafaray.org"
+SRC_URI="https://github.com/YafaRay/Core/archive/v${PV}.tar.gz -> ${P}.tar.gz
```

Comment:
Minor nit: could you make both archives at least kinda-resemble upstream directory naming? We generally prefer that tarballs are named the same as directories inside them. However, naming them just 'Core' and 'Blender-Exporter' would suck for uniqueness, so maybe 'YafaRay-Core-...' and 'YafaRay-Blender-Exporter-...'?